src/main/java/cis2910c/Main.java

```diff
@@ -0,0 +1,278 @@
+			String conditionDisability = input.nextLine();
+			
+			System.out.println("\nEnter disability rating percent:");
+			int percentDisability = input.nextInt();
+			
+			disabilityList.add(new DisabilityRating(percentDisability, conditionDisability));
+			
+			System.out.println("\nAdd another disability? "+
+								"\n1.Yes" +
+								"\n2.No");
+			
+			if(input.nextInt() == 2) {
+				finish = true;
+			}
+					
+		}
+		
+		/*
+		 *  END
+		 *   
+		 */
+		
+		//method to check if bilateral
+		bilateralFactor(disabilityList);
+		
+		//method to sort from highest percentage to lowest
+		sortDisabilityPercentage(disabilityList);
+		
+		System.out.println("\nYour disabilities:");
+		//output percentage and condition
+		for(DisabilityRating d : disabilityList) {
+			System.out.printf("%s%% ", d.getPercent());
+			System.out.printf("%s\n", d.getCondition());
+		}
+		
+		//output disability rating
+		System.out.printf("\nYour Disability Rating: %s%%", calculateDisabilitRating(disabilityList));
+		System.out.println("\nYour monthly payment: $" + CompenstationRates.Basic.DependentStatus.Alone_No_Depends.userRating(calculateDisabilitRating(disabilityList)));
+	}
+	
+	public static void specialRates() {
+		System.out.println("\n" + CompenstationRates.Special.DependentStatus.Alone_No_Depends.userRating("SMC_N_1_2"));
+	}
+	
+	//method to get disability rating from input
+	public static int calculateDisabilitRating(ArrayList<DisabilityRating> disabilityRating) {
+		
+		double efficiencyRating = 100.00;
+		double newEfficiencyRating = efficiencyRating;
+		double newDisabilityRating = 0;
+		int result = 0;
+		
+		//loop through disability ratings and run formula to calculate the efficiency rating
+		for(int i = 0; i < disabilityRating.size(); i++) {
+			
+			newDisabilityRating = Math.round(newEfficiencyRating * ((double) disabilityRating.get(i).getPercent() / 100));
+			newEfficiencyRating = newEfficiencyRating - newDisabilityRating;
+		}
+		
+		//get final efficiency rating
+		newEfficiencyRating = efficiencyRating - newEfficiencyRating;
+		
+		//VA math has disability rating round to the nearest 10th
+		result = roundNearestTenth((int) newEfficiencyRating);
+		
+		return result;
+	}
+	
+	//method to test the disabilityList if disability is bilateral
+	public static void bilateralFactor(ArrayList<DisabilityRating> disabilityList) {
+		
+		String disability;
+		ArrayList <String> disabilityCondition = new ArrayList<>();
+		List<ArrayList<DisabilityRating>> bilateral = new ArrayList<ArrayList<DisabilityRating>>();
+		ArrayList<DisabilityRating> listToRemove = new ArrayList<>();
+		ArrayList<DisabilityRating> tempList = new ArrayList<>();
+		
+		//loop and extract the condition as a string then add to disabilityCondition list
+		for(int i = 0; i < disabilityList.size(); i++) {
+			
+			disability = disabilityList.get(i).getCondition().substring(disabilityList.get(i).getCondition().lastIndexOf(" ") + 1);
+			disabilityCondition.add(disability);
+		}
```

Comment:
You are on the correct. This part of the code is only to push the body parts (e.g "Leg", "Arm", etc..) into a new ArrayList. I the case of you example above, if there are two legs and one arm, the loop will push each body part to the"disabilityCondition" ArrayList. It should be:

[0] =>Leg
[1] =>Leg
[2] =>Arm

I still need to do some work on this because it only detects if the substring is the same, not if there is a "right" and "left" condition with the same substring. But, due to time, I only had it detect if something like "Leg" shows up more than once.

The ideal logic would detect if the leg has both "right" and "left", but I currently have it only to detect if a body part shows up twice to test the calculations.